init.lua

```diff
@@ -786,6 +803,10 @@ function Form:show_hud(player, ctx)
     hud_fs.show_hud(player, self, tree)
 end
 
+function Form:set_as_inventory_for(player, ctx)
+    set_inv_form(self, player, ctx or {})
+end
```

Comment:
I wonder whether it would be better to merge `set_inv_form` into this function and make this function preserve the previous `ctx` (if the form is already set as the inventory and if the `ctx` argument isn't present) and `auto_name_id` (even if a different form was previously set).

Reply:
done